config/tslint.base.json

```diff
@@ -37,7 +37,7 @@
     "no-var-keyword": true,
     "object-literal-shorthand": true,
     "only-arrow-functions": [true, "allow-declarations", "allow-named-functions"],
-    "prefer-const": true,
+    "prefer-const": [true, {"destructuring": "all"}],
```

Comment:
prevents warnings when destructuring multiple fields, unless they should all be const.
e.g. 
```
let {foo, bar} = far;
bar += 1;
```